src/app/(registration)/auth-layout.tsx

```diff
@@ -75,17 +81,8 @@ function Layout({ children }: { children: React.ReactNode }) {
   }, [clerkRef, checkForClerkElement]);
 
   return (
-    <RegistrationLayout
-      asideSlot={
-        <OakBox $maxWidth="all-spacing-21">
-          <CMSImage
-            image={getIllustrationAsset("auth-acorn")}
-            $width="100%"
-            $objectFit="contain"
-          />
-        </OakBox>
-      }
-    >
+    <RegistrationLayout asideSlot={asideSlot}>
+      {headerSlot}
```

Comment:
Ergh, I hate adding new slots but I needed a way to add the inline banner outside the box-shadow wrapper